kafka-connect-s3/docs/s3_connector.rst

```diff
@@ -95,6 +95,21 @@ NO compatibility, BACKWARD compatibility, FORWARD compatibility and FULL compati
 
 Schema evolution in the S3 connector works in the same way as in the `HDFS connector <../../../connect-hdfs/docs/hdfs_connector.html#schema-evolution>`_.
   
+Automatic Retries
+-----------------
+Users' environments might experience network partitioning, reach AWS throttling limits etc.
+In order to have a robust system it makes sense to have a retry policy, however, it's important
+to avoid a thundering herd problem.
```

Comment:
Maybe something a bit closer to what's in ES docs:
> The S3 connector may experience problems writing to the S3 bucket, such as during network partitions, interruptions, or even AWS throttling limits. In many cases, the connector will retry the request a number of times before failing. To prevent from further overloading the network or S3 service, the connector uses an exponential backoff technique to give the network and/or service time to recover.  The technique adds randomness, called jitter, to the calculated backoff times to prevent a thundering herd, where large numbers of requests from many tasks are submitted concurrently and overwhelm the service. Randomness spreads out the retries from many tasks and should reduce the overall time required to complete all outstanding requests compared to simple exponential backoff. The goal is to spread out the requests to S3 as much as possible.

Reply:
Sure, will borrow this section as much as applicable to S3 connector.

Reply:
✅